modules/ztp-configuring-cluster-policies.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: CONCEPT
+[id="ztp-configuring-cluster-policies_{context}"]
+= Configuring managed clusters with policies and PolicyGenTemplate resources
+
+Zero touch provisioning (ZTP) uses {rh-rhacm-first} to configure clusters by using a policy-based governance approach to applying the configuration.
+
+The policy generator or `PolicyGen` is a plug-in for the GitOps operator that enables creation of {rh-rhacm} policies from a concise template. The tool can combine multiple CRs into a single policy, and you can generate multiple policies that apply to various subsets of clusters in your fleet.
+
+The following diagram shows how the policy generator interacts with GitOps and {rh-rhacm} in the configuration phase of cluster deployment.
+
+image::217_OpenShift_Zero_Touch_Provisioning_updates_1022_3.png[Policy generator]
+
+For large fleets of clusters, it is typical for there to be a high-level of consistency in the configuration of those clusters. The following recommended structuring of policies combines configuration CRs to meet several goals:
+
+* Describe common configurations once and apply to the appropriate set of clusters.
+
+* Minimize the number of policies being maintained and managed.
+
+* Support flexibility in common configurations for cluster variants.
+
+.Recommended PolicyGenTemplate policy categories
+[cols="1,5", width="100%", options="header"]
+|====
+|Policy category
+|Description
+
+|Common
+|A policy that exists in the `Common` category is applied to all clusters in the fleet. Use common `PolicyGenTemplate` CRs to apply common installation settings across all cluster types.
+
+|Groups
+|A policy that exists in the `Groups` category is applied to a group of clusters in the fleet. Use group `PolicyGenTemplate` CRs to manage specific aspects of single-node, three-node, and standard cluster installations. Cluster groups can also follow geographic region, hardware variant, etc.
+
+|Sites
+|A policy that exists in the `Sites` category is applied to a specific cluster site. Any cluster
+can have its own specific policies maintained.
+|====
```

Comment:
Some additional text to help guide users between sites and groups:

For scalability and to reduce the complexity of managing configuration across the fleet of clusters, it is recommended to manage configuration CRs with as much commonality as possible. Where possible, CRs should be included in the fleet-wide common policy. The next preference is to create logical groupings of clusters which allow as much of the remaining configuration to be managed under a group policy. When configuration is truly site-specific you may use ACM hub side templating to inject site-specific data into a common or group policy. Alternatively, a site policy may be used.